WooCommerce/Classes/POS/Presentation/Item Selector/ItemList.swift

```diff
@@ -26,9 +27,15 @@ private struct ItemListRow: View {
             }, label: {
                 SimpleProductCardView(product: product)
             })
-        case let .parentProduct(parentProduct):
+        case let .variableParentProduct(parentProduct):
             NavigationLink(value: item) {
-                ParentProductCardView(parentProduct: parentProduct)
+                ParentProductCardView(name: parentProduct.name,
+                                      imageSource: parentProduct.productImageSource,
+                                      detailView: {
+                    Text(Localization.variationsAvailable)
+                        .foregroundStyle(Color.posSecondaryText)
+                        .font(.posBodyRegular)
+                })
```

Comment:
It would also be fine to make this a `VariableParentProductCardView`, and continue to pass through the whole `parentProduct` here. Since we have to define the detail view as a closure, it's not quite as nice as the `ChildItemList` view above, where we simply pass through title string.

Reply:
The image + title design seems pretty fixed for parent product types, I think it's nice sharing most of the view except the detail view. However, if the future design has more differences than the detail view, let's move to a separate view per parent product case for sure.